src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,66 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.